### DISCUSSION TITLE
Inventory Bucket still empty?

### DISCUSSION THREAD

**Discussion by Konkrad** [Element84/earth-search#32] [General]
Hello,
in this information update (https://us13.campaign-archive.com/?u=a7a7fcb1ce46c4d001fc76289&id=c5e6cf4785) it is mentioned that the s3 inventory data will end up in the bucket `e84-earth-search-sentinel-data-inventory`. For me the bucket still appears to be empty. When do you plan to activate this feature? 
> The s3 inventory files for the new bucket are in a bucket e84-earth-search-sentinel-data-inventory

thank you

**Comment by philvarner** [comment]
Thank you for bringing this issue to our attention! It should be being populated as of a few weeks ago.
